### PR TITLE
[docs] flushEvents parameters section fix

### PR DESCRIPTION
**Pull request by PaulMcClernan**
I think this should fix the parameters section

### CODE REVIEW

**Comment by PaulMcClernan** [comment]
@seaniepie I'm just randomly picking doc items that I use often that are on your list.

**Comment by seaniepie** [comment]
@PaulMcClernan This is great. I've updated the doc again to highlight the ones that are done so far. If you don't still have a live link to it via dropbox I'll resend the url.

**Comment by PaulMcClernan** [comment]
I still have the link in the original closed pull request.

File: docs/dictionary/function/flushEvents.lcdoc [hunk 1]
```diff
@@ -21,13 +21,18 @@ Example:
...
+-  all: ignore all waiting events
+-  mouseDown: ignore mouse presses
+-  mouseUp: ignore mouse releases
+-  keyDown: ignore keypresses
+-  keyUp: ignore key releases
+-  autoKey: ignore key repeats
+-  disk: ignore disk-related events
+-  activate: ignore windows being brought to the front
+-  highLevel: ignore Apple Events (on Mac OS and OS X systems)
+-  system: ignore operating system events
```
Comment (livecodeali) [inline docs/dictionary/function/flushEvents.lcdoc]: If you change the type here to `(enum)`, then you can remove the `One of the following:` line (this is automatically added for enum params). Each of the actual options should be enclosed in double quotes - 
- "all": ...
- "mouseDown": ...
- "mouseUp": ...

etc

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 9ae4696

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 9ae469647b41daab8394323eafebdacb4d43b339

**Comment by livecode-vulcan** [comment]
😎 test success 9ae469647b41daab8394323eafebdacb4d43b339
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/467)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/472)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/467)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/467)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/473)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/467)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/467)

**Pull request merged**